GlobalData.txt

```diff
@@ -18,4 +18,8 @@ Referrers: http://localhost:3000/
 Activation date: Jun 3, 2015, 1:18:00 PM
 Activated by: aniltilanthe92@gmail.com (you)
 
-_______________________________________________________________________
+____________________________________________________________________
+Client ID for google sign in button
```

Comment:
Why have you pushed this ?
This is not needed right ? We have used the first one which is existing already.
Can you write all the changes in a paper & then do them each. So that you do not forget anything.